torch/csrc/jit/operator_upgraders/version_map.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <string>
+#include <vector>
+#include <unordered_map>
+
+namespace torch {
+namespace jit {
+
+using UpgraderDB = std::vector<std::pair<int, std::string>>;
```

Comment:
I'm still thinking if we should make it more look-up friendly, as parsing string on the fly is not necessary and time consuming.